### ISSUE TITLE
Use debian stable instead of testing?

### ISSUE THREAD

**Issue by xuhdev** [spiroid/docker-mozilla-syncserver#1]
Is there a reason to run debian testing? It does not seem stable enough for a long term server.

**Comment by spiroid** [comment]
@xuhdev : i'm using debian testing on my servers for a few years now and it always gave me the stability i needed with fresh enough packages (newer versions more often).

Would this be an obstacle for you, or a major issue with this image if i keep using a debian testing as base ?

**Comment by xuhdev** [comment]
@spiroid The reason to use a stable release of Debian on a server has a few advantages:
- stability. Since Debian testing keeps upgrading the software in the system, a single upgrade may break the compatibility. Let's say you have tested Mozilla sync in Debian testing at this moment, sometime later the system upgrades and there comes an incompatibility, for which Mozilla cannot fix (this can be a long time depends on the problem). People with new installation of this docker will not get it work. On the other hand, I don't think the version of Debian stable is too old for Mozilla sync, as Debian stable already has a relatively faster releasing cycle than some others, e.g. RHEL/CentOS. By using Debian stable, we don't lose any Mozilla sync's functionality, but can have more peace in mind.
- Security update. From [Debian wiki](https://wiki.debian.org/DebianTesting): `Compared to stable and unstable, next-stable testing has the worst security update speed. Don't prefer testing if security is a concern.`

I understand that you haven't encountered any problem in the past few years, but server stability and security is a matter of probability -- you can only reduce the risk, but there is no way to eliminate them. And not encountering any problem does not mean the risk is low.

In conclusion, I suggest to move to Debian stable (jessi at this moment). I would be very willing to contribute if it's moved to a more stable base, but the current situation really makes me hesitate.
